tests/foreman/ui/test_rhc.py

```diff
@@ -65,10 +66,18 @@ def fixture_setup_rhc_satellite(request, module_target_sat, module_rhc_org):
         module_target_sat.cli.Subscription.upload(
             {'file': manifests_path, 'organization-id': module_rhc_org.id}
         )
+        # Enable and sync required repos
+        repo1_id = enable_sync_redhat_repo(REPOS['rhel8_aps'], module_rhc_org.id)
```

Comment:
@jameerpathan111 You have enabled rhel8_bos and rhcsl repos as well, arent you syncing them ?

Reply:
@jyejare they are for satellite itself, and this pr is for making content available to iqe vm's.